### PR TITLE
Add skip-kamal option to the command line docs

### LINKED ISSUE #53051: skip-kamal option should be added to CLI docs
https://edgeguides.rubyonrails.org/command_line.html does not list `--skip-kamal` as an option.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #53051

### CODE REVIEW

File: guides/source/command_line.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ If you wish to skip some files from being generated or skip some libraries, you
 | `--skip-javascript`     | Skip JavaScript files                                    |
 | `--skip-hotwire`        | Skip Hotwire integration                                 |
 | `--skip-jbuilder`       | Skip jbuilder gem                                        |
+| `--skip-kamal`          | Skip Kamal gem                                           |
```
Comment (p8) [inline guides/source/command_line.md:65]: The official name of the gem is lowercase: https://rubygems.org/gems/kamal (just like jbuilder):
```suggestion
| `--skip-kamal`          | Skip kamal gem                                           |
```

**Review by p8 — CHANGES_REQUESTED** [review]

File: guides/source/command_line.md (hunk 1 above)
Comment (jcoyne) [reply guides/source/command_line.md:65]: It is capitalized here https://kamal-deploy.org

Comment (p8) [reply guides/source/command_line.md:65]: I think the name of the gem is used. 
Jbuilder is capitalized here as well, but the gem name still is jbuilder:
https://github.com/rails/jbuilder?tab=readme-ov-file#contributing-to-jbuilder

Comment (zzak) [reply guides/source/command_line.md:65]: I could have sworn we discussed maintaining this list recently :thinking:

Comment (rafaelfranca) [inline guides/source/command_line.md:65]:
```suggestion
| `--skip-kamal`          | Skip Kamal files                                         |
```

Comment (rafaelfranca) [reply guides/source/command_line.md:65]: It isn't only the gem that is added

**Comment by skipkayhil** [comment]
Thank you for the pull request, but we're going to go another direction: #53924

**Pull request closed without merging**